icu4c/source/data/mappings/ibm-1388_P100-2024.ucm

```diff
@@ -1,16 +1,9 @@
 # Copyright (C) 2016 and later: Unicode, Inc. and others.
 # License & terms of use: http://www.unicode.org/copyright.html
-# *******************************************************************************
-# *
-# *   Copyright (C) 1995-2001, International Business Machines
-# *   Corporation and others.  All Rights Reserved.
-# *
-# *******************************************************************************
 #
-# File created by rptp2ucm (compiled on Dec 18 2001)
```

Comment:
Is the new version not created by rptp2ucm?

Reply:
The group who managed CDRA is gone, and the data was migrated from Notes database to something else. My understanding is that it's no longer handle code pages in the way they previously did.

Reply:
Ok, but the following line is updated, referring to different versions of .RPMAP* & .TPMAP*.
It seems weird to just delete the first half of the sentence here.
Did someone run the tool, or did they fiddle manually with the mappings?

Reply:
Someone in GCoC Tokyo made the change - probably manually. I had to fix some issues on top of it (including state table).